R/plotDataSetList.R

```diff
@@ -1059,7 +1075,7 @@ Plot.FV.ECDF_AUC.DataSetList <- function(dsList, rt_min = NULL, rt_max = NULL, r
   if (is.null(rt_min)) rt_min <- min(rt)
   if (is.null(rt_max)) rt_max <- max(rt)
 
-  rt_seq <- seq_RT(rt, from = rt_min, to = rt_max, by = rt_step)
+  rt_seq <- seq_RT(rt, from = rt_min, to = rt_max, by = rt_step) %>% round
```

Comment:
perhaps we should put `round` function within `seq_RT` as we only want as output an array of integers?

Reply:
That is a good idea, I will add this